src/redis.ts

```diff
@@ -1,16 +1,13 @@
 import Redis from "ioredis"
+import { baseLogger } from "./logger"
 
 let connectionObj = {}, natMap = {}
 
 if(process.env.LOCAL === 'true') {
-  const REDIS_0_INTERNAL_IP = `${process.env.REDIS_0_INTERNAL_IP}:6379` 
-  const REDIS_1_INTERNAL_IP = `${process.env.REDIS_1_INTERNAL_IP}:6379`
-  const REDIS_2_INTERNAL_IP = `${process.env.REDIS_2_INTERNAL_IP}:6379`
+  const REDIS_0_INTERNAL_IP = `${process.env.REDIS_0_INTERNAL_IP}:6379`
   
   natMap = {
-    [REDIS_0_INTERNAL_IP]: { host: process.env.REDIS_0_DNS, port: process.env.REDIS_0_PORT },
-    [REDIS_1_INTERNAL_IP]: { host: process.env.REDIS_1_DNS, port: process.env.REDIS_1_PORT },
-    [REDIS_2_INTERNAL_IP]: { host: process.env.REDIS_2_DNS, port: process.env.REDIS_2_PORT }
+    [REDIS_0_INTERNAL_IP]: { host: process.env.REDIS_0_DNS, port: process.env.REDIS_0_PORT }
```

Comment:
could you explain this syntax here?
`{[ip]: {}}` 
I'm not familiar with the use of square bracket for the key of an object.

Reply:
Check [this](https://stackoverflow.com/a/11508490/1124303) out